docs/lessons/buttons.rst

```diff
@@ -0,0 +1,188 @@
+Buttons
+-------
+
+So far we have created code that makes the device do something. This is called
+*output*. However, we also need the device to react to things. Such things are
+called *input*.
```

Comment:
Obviously English is not my primary language, but `such things are` in plural and then `input` in singular doesn't feel quite right.